client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -49,7 +49,6 @@ export const UnwrappedUpgradePlan: React.FunctionComponent< UpgradePlanProps > =
 		coupon: undefined,
 		planSlugs: [ visiblePlan ],
 		siteId: site.ID,
-		storageAddOns: null,
```

Comment:
NOTE:

Since the default value of `reflectStorageSelectionInPlanPrices` in the `usePricingMetaForGridPlans` hook is `false`, explicitly setting a `falsey` value is no longer necessary. It might, however, still be worthwhile to encourage this if we want other developers to be aware of the `reflectStorageSelectionInPlanPrices`  flag.

I'm inclined to remove the param, if folks feel otherwise I'm happy to add back an explicit `reflectStorageSelectionInPlanPrices: false` argument.